ui/changes/src/graph.js

```diff
@@ -21,80 +21,36 @@ function generateData(count, yrange) {
   return series;
 }
 
-function rerender(connections, teamComponentMapping) {
+async function generateData() {
+  let map = await getComponentRegressionMap();
+  let return_map = []
+  Object.keys(map)
+    .slice(0, 10)
+    .forEach(element => {
+      let values = {}
+      values.data = []
+      values.name = element
+      for (const value in map[element]){
+        let obj = {}
+        obj.x = value
+        obj.y = Math.trunc(map[element][value] * 100)
```

Comment:
TIL: [`Math.trunc()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/trunc) is a thing!!

```
/giphy "the more you know"
```